Sources/NIO/Bootstrap.swift

```diff
@@ -705,3 +705,88 @@ public final class DatagramBootstrap {
         }
     }
 }
+
+/// A `PipeBootstrap` is an easy way to bootstrap a `PipeChannel` which uses two (uni-directional) UNIX pipes
+/// and makes a `Channel` out of them.
+public final class PipeBootstrap {
+    private let group: EventLoopGroup
+    private var channelInitializer: ((Channel) -> EventLoopFuture<Void>)?
+    @usableFromInline
+    internal var _channelOptions = ChannelOptions.Storage()
+
+    /// Create a `PipeBootstrap` on the `EventLoopGroup` `group`.
+    ///
+    /// - parameters:
+    ///     - group: The `EventLoopGroup` to use.
+    public init(group: EventLoopGroup) {
+        self.group = group
+    }
+
+    /// Initialize the connected `PipeChannel` with `initializer`. The most common task in initializer is to add
+    /// `ChannelHandler`s to the `ChannelPipeline`.
+    ///
+    /// The connected `Channel` will operate on `ByteBuffer` as inbound and outbound messages. Please note that
+    /// `IOData.fileRegion` is _not_ supported for `PipeChannel`s because `sendfile` only works on sockets.
+    ///
+    /// - parameters:
+    ///     - handler: A closure that initializes the provided `Channel`.
+    public func channelInitializer(_ handler: @escaping (Channel) -> EventLoopFuture<Void>) -> Self {
+        self.channelInitializer = handler
+        return self
+    }
+
+    /// Specifies a `ChannelOption` to be applied to the `PipeChannel`.
+    ///
+    /// - parameters:
+    ///     - option: The option to be applied.
+    ///     - value: The value for the option.
+    @inlinable
+    public func channelOption<Option: ChannelOption>(_ option: Option, value: Option.Value) -> Self {
+        self._channelOptions.append(key: option, value: value)
+        return self
+    }
+
+    /// Create the `PipeChannel` with the provided input and output file descriptors..
+    ///
+    /// - parameters:
+    ///     - inputDescriptor: The _Unix file descriptor_ for the input (ie. the read side).
+    ///     - outputDescriptor: The _Unix file descriptor_ for the input (ie. the write side).
+    /// - returns: an `EventLoopFuture<Channel>` to deliver the `Channel` immediately.
```

Comment:
The word "immediately" here is not accurate: this future may take a while to be delivered if the initializer delays setup.

Reply:
yeah, sorry, I copy & pasted that from our existing docs, will fix that.